### PR TITLE
VEN-1276  &  VEN-1314 | Remove Old Harbors App

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Update all queries, utils, fixtures, and tests to reflect the new changes in the GraphQL schema
- Ditch `servicemapId` in fetching harbors and use `harborId` instead

### Automated tests :gear:️
- Updated all related tests and snapshots

### Manual testing :construction_worker_man:
- Try all the features in the application, everything should work as expected

### CODE REVIEW

File: src/features/berthOffer/__tests__/__snapshots__/utils.test.ts.snap [hunk 1]
```diff
@@ -1,77 +1,7 @@
...
-exports[`utils getAllPiersIdentifiers should return pier tabs 1`] = `
-Array [
-  Object {
-    "disabled": false,
-    "label": "pier",
-    "value": "pier",
-  },
-]
-`;
+exports[`utils getAllPiersIdentifiers should return pier tabs 1`] = `Array []`;
```
Comment (TatuArvela) [inline src/features/berthOffer/__tests__/__snapshots__/utils.test.ts.snap:3]: These snapshots don't look right

**Review by TatuArvela — CHANGES_REQUESTED** [review]
Spotted one oddity in the snapshots you should check. Excellent work! 🥇

File: src/features/berthOffer/__tests__/__snapshots__/utils.test.ts.snap (hunk 1 above)
Comment (Nizar-Rahme) [reply src/features/berthOffer/__tests__/__snapshots__/utils.test.ts.snap]: Good catch, fixed!

**Review by TatuArvela — APPROVED** [review]
:shipit:

**Pull request merged**